staging/src/k8s.io/kubectl/pkg/cmd/autoscale/autoscale.go

```diff
@@ -118,8 +118,6 @@ func NewCmdAutoscale(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *
 	// bind flag structs
 	o.RecordFlags.AddFlags(cmd)
 	o.PrintFlags.AddFlags(cmd)
-	cmd.Flags().String("generator", "horizontalpodautoscaler/v1", i18n.T("The name of the API generator to use. Currently there is only 1 generator."))
```

Comment:
/hold
that's too soon, the flag was deprecated in https://github.com/kubernetes/kubernetes/pull/92998 even though it's not used and not functional [deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/) clearly states the time line and we need to keep it.

Reply:
So per deprecation policy (sorry, I've missed that when opening the issue!)
```
Rule #5b: CLI elements of admin-facing components (e.g. kubelet) must function after their announced deprecation for no less than:
    GA: 6 months or 1 release (whichever is longer)
```

As that PR was merged for v1.20 (which was released on Dec/2020) we should wait until jun/2020 to get this merged right?

Probably the timeline for v1.22 (that's going to be released in...Jun/2020 I guess) would allow us to merge this, so the good approach here is to wait until v1.21 is released and then merge this, is this right?

Thanks @soltysh

Reply:
1.21 will release on April，2021.
need we add milestone 1.22 for this PR?
/milestone 1.22

Reply:
@pacoxu we are almost at code freeze anyway, so we can wait :)